picard/script/parser.py

```diff
@@ -239,6 +247,17 @@ def read(self):
                 self._x += 1
         return ch
 
+    def read_multi(self, count):
+        text = ch = self.read()
```

Comment:
I guess you need to test result here, because the script could be just `\\u`, it should also raise eof in this case (add a test for it).

Reply:
Right, this was not handled properly. Fixed and extended the tests.